Extensions.Caching.PostgreSql/DatabaseOperations.cs

```diff
@@ -43,6 +44,7 @@ public DatabaseOperations(IOptions<PostgreSqlCacheOptions> options, ILogger<Data
             SqlCommands = new SqlCommands(cacheOptions.SchemaName, cacheOptions.TableName);
 
             this._logger = logger;
+            this._disableUpdateOnGetCacheItem = cacheOptions.DisableUpdateOnGetCacheItem;
```

Comment:
Maybe negate the variable name to make the if-statements easier to read?
```suggestion
            this._updateOnGetCacheItem = !cacheOptions.DisableUpdateOnGetCacheItem;
```